### ISSUE TITLE
ReferenceError: web3 is not defined

### ISSUE THREAD

**Issue by amachino** [ensdomains/ensjs-v2#14]
```
node_modules/ethereum-ens/index.js:416
    var registryContract = web3.eth.contract(registryInterface);
                           ^

ReferenceError: web3 is not defined
    at new ENS (/Users/akinori/ens/node_modules/ethereum-ens/index.js:416:28)
    at Object.<anonymous> (/Users/akinori/ens/index.js:5:13)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
    at Function.Module.runMain (module.js:682:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:613:3
```

**Comment by amachino** [comment]
https://github.com/Arachnid/ensjs/commit/0b7910046a2136aa272d48a2e03d28fab9304e6e#r26606861

**Comment by Arachnid** [comment]
Fixed in #13.
